### PR TITLE
Add OriginCode

### PR DESCRIPTION
**Pull request by OriginCode**

### CODE REVIEW

File: data/people.yml [hunk 1]
```diff
@@ -124,6 +124,12 @@
   description: Faithful Listener | Cool-headed Geek | Mind Swimmer | Ever-Loner | THE Software Magician
   longdesc: Long desc is not necessary for a guy like me who lives in the void.
   rurl:
+-
+  fullname: Jack Wu
+  username: OriginCode
```
Comment (MingcongBai) [inline data/people.yml:129]: Your name has been placed at an incorrect location. Names are sorted alphabetically according to `username`.

Comment (OriginCode) [reply data/people.yml]: 4af9a92

**Comment by OriginCode** [comment]
> Please also add a new page under `content/people/` with name `OriginCode.html` or `OriginCode.md`.
> 
> You may want to also add these lines to your page:
> 
> ```
> ---
> layout: people
> url: /people/~OriginCode/
> ---
> ```

4a5991b

**Comment by MingcongBai** [comment]
Thank you and welcome to the gang!

**Pull request merged**